system/infra-monitoring/templates/_prometheus-infra-collector.yaml.tpl

```diff
@@ -432,3 +432,23 @@
     - target_label: __address__
       replacement: vrops-exporter:9160
 {{- end }}
+
+#exporter is leveraging service discovery but not part of infrastructure monitoring project itself.
+{{- $values := .Values.esxi_exporter -}}
+{{- if $values.enabled }}
+- job_name: 'esxi-config'
+  scrape_interval: {{$values.scrapeInterval}}
+  scrape_timeout: {{$values.scrapeTimeout}}
+  file_sd_configs:
+      - files :
+        - /etc/prometheus/configmaps/atlas-sd/netbox.json
+  metrics_path: /
+  relabel_configs:
+    - source_labels: [job]
+      regex: vcenter
+      action: keep
+    - source_labels: [server_name]
+      target_label: __param_target
```

Comment:
Only param queried would be `.../?target=vcenter-name`
Is this wanted?

Reply:
yes, this is wanted